src/rt/minfo.d

```diff
@@ -339,8 +353,7 @@ struct ModuleGroup
                 else
                 {
                     auto midx = edges[sp.curMod][sp.curDep];
-                    // if midx is -1, then this isn't part of this DSO.
-                    if (midx != -1 && !bts(reachable, midx))
+                    if (!bts(reachable, midx))
```

Comment:
Yes, I realized this was a remnant from the implementation that still used the `findModule` function. I don't put cross-module dependencies as -1 in the `edges` array, so no point in checking for -1 here.